cmd/livepeer/latency_test.go

```diff
@@ -0,0 +1,334 @@
+//go:build nvidia
+// +build nvidia
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func toSeconds(duration time.Duration) float64 {
+	return float64(duration) / 1000000000.0
+}
+func fromSeconds(seconds float64) time.Duration {
+	return time.Duration(seconds * 1000000000.0)
+}
```

Comment:
@thomshutt I use these helper functions, is there another way that better matches the style of the go code?

Reply:
Nope, seems pretty go-ish!